app/main.ipynb

```diff
@@ -126,19 +148,28 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
-   "outputs": [],
    "source": [
     "array = np.array([2.3, 7.8, 3.2, 1.1, 5.8, 9.5, 17.6, 11.1])\n",
-    "# write your code here"
+    "selected_elements = array[[1, 2, 3, 4, 5]]\n",
+    "print(selected_elements.sum())"
```

Comment:
The sum of the selected elements is 27.4, which does not match the expected result of 27. Please verify the indices used for selection to ensure they match the task requirements.